runtime/ddr/blacklist

```diff
@@ -27,6 +27,44 @@ file:<built-in>
 # zlib defines the type 'Byte' which conflicts with java.lang.Byte.
 file:*/zlib/zconf.h
 
-# Ignore types that are troublesome or not required. 
+# Ignore types that are troublesome or not required.
 type:__gnu*
 type:*__va_list*
+type:*vc_attributes*
+type:*<*
+type:tag*
```

Comment:
what is there is a type starting with tag we don't want to ignore?

Reply:
I think the majority of these are due to this pattern:
```
typedef struct tagSample { /*...*/ } Sample;
```
We will have the information for `Sample`, we don't need to have `tagSample` as well.

If we find we need some type, say `tagBar`, we can either
* blacklist tagA*, tagC*, tagD*, etc.
* expand the pattern language, or
* add the ability to white-list types

Reply:
The problem is that when we find we need some type, its already too late because the core file cannot necessarily be recreated.